src/scss/_profile.scss

```diff
@@ -0,0 +1,108 @@
+body,
+html {
+  height: 100%;
+  margin: 0;
+}
+
+.user-profile-picture {
+  top: 30%;
+  left: 50%;
+  transform: translate(-50%, -50%);
```

Comment:
Using absolute positioning with transform is not a sustainable way to create layouts. Flex or grid positioning are much more reliable for this purpose.